airbyte-integrations/connectors/source-mssql/metadata.yaml

```diff
@@ -18,20 +18,18 @@ data:
   name: Microsoft SQL Server (MSSQL)
   registries:
     cloud:
-      # CI pipeline is broken for mssql
-      dockerImageTag: 3.6.1
       enabled: true
     oss:
-      # CI pipeline is broken for mssql
-      dockerImageTag: 3.6.1
       enabled: true
-  releaseStage: alpha
+  releaseStage: generally_available
   supportLevel: community
   tags:
     - language:java
-    - language:python
   releases:
     breakingChanges:
+      4.0.0:
+        message: "Certified source mssql connector."
```

Comment:
If you say certified here, then you need to change line 25 too

Reply:
might want to write something along the lines of
"We have overhauled our MSSQL source connector and it is now supported by the Airbyte team!  To benefit from  new features, including terabyte-sized table support, reliability improvements, expanded datetime data types, and various bug fixes, please opt in to the 4.0.0 version."